.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Release checklist
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all the important changes in `appendix/changelog.rst`
+- [ ] Set the release version and date in `appendix/changelog.rst`
+- [ ] Set the release version `SACDOCVERSION` in `source/SAC_style.sty`
+- [ ] Set the release version in `source/conf.py`
+- [ ] Add the release information to `source/version.rst`
+- [ ] Upload the PDF to the release
+- [ ] Bump the version number to "x.x-dev" in `source/conf.py` and `source/SAC_style.sty`
```

Comment:
```suggestion
- [ ] Bump the version number to "X.Y-dev" in `source/conf.py` and `source/SAC_style.sty`
```

Reply:
Add bumping SAC version?

Reply:
Please see the updated suggestion above.